pkgs/games/osu-lazer-bin/default.nix

```diff
@@ -0,0 +1,35 @@
+{ appimageTools, lib, fetchurl }:
+
+appimageTools.wrapType2 rec {
+  pname = "osu-lazer-bin";
+  version = "2022.1228.0";
+
+  src = fetchurl {
+    url = "https://github.com/ppy/osu/releases/download/2022.1228.0/osu.AppImage";
```

Comment:
```suggestion
    url = "https://github.com/ppy/osu/releases/download/${version}/osu.AppImage";
```

Reply:
done in 50dbb3001c3fa43e351e717b2b7c4e23c1363149